egs/wsj/s5/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -131,7 +131,7 @@ fi
 if [ $stage -le 13 ]; then
   # Get the alignments as lattices (gives the chain training more freedom).
   # use the same num-jobs as the alignments
-  steps/align_fmllr_lats.sh --nj 100 --cmd "$train_cmd" ${lores_train_data_dir} \
+  steps/align_fmllr_lats.sh --nj $nj --cmd "$train_cmd" ${lores_train_data_dir} \
```

Comment:
@danpovey @johnjosephmorgan  I know there was a conversation on this and I think at the end the sentiment was to leave 100 here?